docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -28,19 +28,37 @@ Combobox is composed of three main parts:
 #### Trigger
 
 The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
-
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+initiate option selection. When implementing Combobox, you may rely on the
+default trigger or, you may pass in a custom activator.
+
+The default trigger is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing a
+label that reads "Select" and an `add` Icon suffix. The label can be customized
```

Comment:
I wonder if we could clarify things with some more specificity

it's actually a Chip `heading` that gets the "Select" value. I know that's a bit pedantic, but since Chip literally does have a `label` prop it might be confusing to be using these terms interchangeably

I get that we call it label because the prop to the `Combobox` is `label`... but maybe we change that? 

also we don't do anything with `label` if they have a custom activator and we don't really communicate the fact that it only shows up if you use the default ComboboxTrigger. some naming or prop documentation might be able to make that clearer

-------

**aside**

the fact that `label` can be an empty string still doesn't sit right with me. it results in this DOM

<img width="1421" alt="image" src="https://github.com/GetJobber/atlantis/assets/11843215/a65bd747-1dd0-460c-92d2-cff8f03271f8">

where we are rendering an empty `span`

the reason it happens is because of this `label={hasSelection ? selectedLabel : ""}`

and we have to do that because Chip has `label` as required so we're skirting around that by using an empty string. feels like we're misusing Chip here, either that or Chip shouldn't be this way. maybe instead Chip says I need at least a `heading` or a `label` - I don't care which but it can't be neither

sorry this has gone off the rails, but I do feel like it's worth considering now while we have both this and Chip in progress!

Reply:
I spoke with Scott briefly this morning about all this.
What do we think about changing `label` in Combobox to be called something entirely different.  

Maybe I'm making way for a use case that will never come up but what if down the line, Combobox's default trigger is no longer a `Chip`.  If it's a `Button`, `label` still does make sense.  If we rename Combobox's `label` prop to something entirely different and specific to Combobox, does that avoid confusion between Combobox props and Chip props?

Reply:
Ok I actually changed the wording in these sections completely, to not reference Chip or Button and just keep it high level enough that if the default activator changed, it wouldn't break our docs.

Your points about the prop name `label` and what Chip is doing is valid, lets bring that convo out of this PR